critiquebrainz/frontend/views/test/test_review.py

```diff
@@ -61,9 +63,37 @@ def test_browse(self):
 
     def test_entity(self):
         review = self.create_dummy_review()
+
+        # test review
         response = self.client.get("/review/%s" % review["id"])
         self.assert200(response)
         self.assertIn(self.review_text, str(response.data))
+        old_text = review["text"]
+
+        # test revisions
+        updated_text = "The text has now been updated"
+        updated_rating = 4
+        db_review.update(
+            review_id=review["id"],
+            drafted=False,
+            text=updated_text,
+            rating=updated_rating,
+            language=review["language"],
+            is_draft=review["is_draft"],
+        )
+
+        # test updated text and rating
+        response = self.client.get("/review/{}/revisions/2".format(review["id"]))
+        self.assert200(response)
+        self.assertIn(updated_text, str(response.data))
+        rating_html = 'input type="number" class="rating" id="rating" value={}'
```

Comment:
this is a bit too clumsy for my tastes. You can get the review variable from the jinja context and check your stuff there.

```suggestion
      review = self.get_context_variable('review')
      self.assertEqual(review['rating'], 4)
```

and so on.

Reply:
@paramsingh Done! Have a look.